lib/galaxy/webapps/galaxy/api/histories.py

```diff
@@ -77,13 +77,14 @@
     example="latest",
 )
 
-
-class HistoryFilterQueryParams(FilterQueryParams):
-    order: Optional[str] = OrderParamField(default_order="create_time-dsc")
-
-
-class HistoryIndexParams(HistoryFilterQueryParams):
-    all: Optional[bool] = False
+AllHistoriesQueryParam = Query(
+    default=False,
+    title="All Histories",
+    description=(
+        "Whether all histories from other users in this Galaxy should be included."
+        " Only admins are allowed to query all histories."
```

Comment:
Intentional space in front of `Only`?

Reply:
Yes, this is [string literal concatenation](https://docs.python.org/3/reference/lexical_analysis.html#string-literal-concatenation) in Python, it is basically just a long line, but for me, it looks cleaner than using backslashes.

Reply:
Most trivial of nits, and maybe addressing Sam's question - maybe easier to read at a glance with the space at the end of the previous string?

Reply:
Oh, I see now! sure I can put the space at the end :laughing: